examples/NAT.cs

```diff
@@ -1,379 +0,0 @@
-/*
```

Comment:
On second thoughts, and having just read more carefully the implementation of the `NATBase` class, we could update the example in the front README to point to the functions [handlePacket](https://github.com/niksu/pax/pull/12/files#diff-3465279ee2070c9b5f37aefb4a0d1cfcR80) and [OutsideToInside](https://github.com/niksu/pax/pull/12/files#diff-3465279ee2070c9b5f37aefb4a0d1cfcR96) which succinctly give the core logic in the much-improved version of the NAT you developed.